File: 07-contactWithAndroid/01-Salutator/app/src/main/java/com/example/salutator/MainActivity.java
package com.example.salutator;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void greet(View v)
    {
        EditText etName = findViewById(R.id.etName);
        String name = etName.getText().toString();
        if(name.isEmpty())
        {
            Toast.makeText(this,
                    "Please, enter your name",
                    Toast.LENGTH_SHORT).show();
        }
        else
        {
            Toast.makeText(this,
                    "Hello, " + name,
                    Toast.LENGTH_SHORT).show();
        }
    }
}

File: 10-intentsAndPermissions/11-Exercise/app/src/main/java/com/example/exercise/MainActivity.kt
package com.example.exercise

import android.content.Context
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
import java.lang.Exception
import androidx.core.app.ComponentActivity.ExtraData
import androidx.core.content.ContextCompat.getSystemService
import android.icu.lang.UCharacter.GraphemeClusterBreak.T
import java.io.*


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        saveBtn.setOnClickListener {

            if(nameET.text.isNotEmpty() && mailET.text.isNotEmpty())
            {
                try {
                    val stream: FileOutputStream = openFileOutput(
                        "data.txt", Context.MODE_APPEND
                    )

                    val file = PrintWriter(stream)
                    file.appendln("${nameET.text} - ${mailET.text}")
                    file.close()
                    nameET.text.clear()
                    mailET.text.clear()
                } catch (e: Exception) {
                    Toast.makeText(this, "ERROR: " + e.stackTrace, Toast.LENGTH_SHORT).show()
                }
            }
            else
            {
                Toast.makeText(this, "Enter name and mail", Toast.LENGTH_SHORT).show()
            }
        }

        showBtn.setOnClickListener {
            val myIntent = Intent(this, People::class.java)
            startActivity(myIntent)
        }
    }
}
